src/Navigation/Navigation.Xamarin.Forms/Service/NavigationService.show.cs

```diff
@@ -196,45 +196,31 @@ internal async Task<TViewModelInterface> InitializeViewModelInternal<TViewModelI
         {
             if (!TryGetRegistration(
                 typeof(TViewModel),
-                out var registrationInfo))
+                out var registration))
             {
                 // todo: set exception message
                 throw new InvalidOperationException();
             }
 
 
-            if (registrationInfo.ViewType.IsAssignableFrom(typeof(TPage)))
+            if (registration.ViewType.IsAssignableFrom(typeof(TPage)))
             {
                 throw new InvalidOperationException(
-                    $"Non-assignable view type specifications for ViewModel '{typeof(TViewModel).Name}' - registered: '{registrationInfo.ViewType.Name}', generic parameter: {typeof(TPage).Name}");
+                    $"Non-assignable view type specifications for ViewModel '{typeof(TViewModel).Name}' - registered: '{registration.ViewType.Name}', generic parameter: {typeof(TPage).Name}");
             }
 
 
             Page view;
 
-            if (Configuration.CachePageInstances)
+            if (Configuration.CacheContent)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)